packages/cli/lib/project-generator.js

```diff
@@ -55,6 +56,24 @@ module.exports = class ProjectGenerator extends BaseGenerator {
       const isValid = utils.validate(this.args[0]);
       if (typeof isValid === 'string') throw new Error(isValid);
     }
+
+    this.setupRenameTransformer();
+  }
+
+  /**
+   * Registers a Transform Stream with Yeoman. Removes `.template` extension
+   * from files that have it during project generation.
+   */
+  setupRenameTransformer() {
+    this.registerTransformStream(
+      rename(function(file) {
```

Comment:
Is there a function that can do this for us without having to import another package? I guess it's not a huge deal regardless though